pkgs/development/python-modules/scikit-hep-testdata/default.nix

```
@@ -0,0 +1,52 @@
+{ lib
+, fetchFromGitHub
+, python
+, buildPythonPackage
+, importlib-resources
+, pyyaml
+, requests
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "scikit-hep-testdata";
+  version = "0.4.2";
+
+  # fetch from github as we want the data files
+  src = fetchFromGitHub {
+    owner = "scikit-hep";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-5ssJztfWC2FC/byoB/e9H1e0iml7gwjsEpWHFdgb+F8=";
+  };
+
+  # https://github.com/scikit-hep/scikit-hep-testdata/issues/59
+  postPatch = ''
+    echo "version = '${version}'" > src/skhep_testdata/version.py
+  '';
+
+  propagatedBuildInputs = [
+    importlib-resources
+    pyyaml
+    requests
+  ];
+
+  postInstall = ''
+    mv src/skhep_testdata/data/*.root "$out"/${python.sitePackages}/skhep_testdata/data/
+  '';
+
+  doCheck = false; # tests require networking
+  checkInputs = [
+    pytestCheckHook
+  ];
+  preCheck = ''
+    export HOME="$(mktemp -d)"
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/scikit-hep/scikit-hep-testdata";
+    description = "A common package to provide example files (e.g., ROOT) for testing and developing packages against.";
```

No period at the end, see https://nixos.org/manual/nixpkgs/stable/#sec-standard-meta-attributes

```suggestion
    description = "Common package to provide example files (e.g., ROOT) for testing and developing packages against";
```

---

Removed the period, left the article.